examples/streaming/README.md

```diff
@@ -134,28 +153,19 @@ var (
     ReadBufferSize: 512,
     WriteBufferSize: 512,
   }
-  myConnConfigurer := func(conn *websocket.Conn) *websocket.Conn {
-    conn.SetReadDeadline(time.Now()+time.Minute*2)
-    return conn
-  }
-  chatterServer = chattersvcsvr.New(chatterEndpoints, mux, dec, enc, eh, myUpgrader, myConnConfigurer)
+  chatterServer = chattersvcsvr.New(chatterEndpoints, mux, dec, enc, eh, myUpgrader, nil, nil, nil, nil)
```

Comment:
I think we should group all the configurers into one struct like we do with endpoints.